rflx/pyrflx/typevalue.py

```diff
@@ -385,6 +385,10 @@ def value(self) -> bytes:
     def bitstring(self) -> Bitstring:
         self._raise_initialized()
         assert self._value is not None
+        size = self.size
+        assert isinstance(size, Number)
+        if size.value == 0:
+            return Bitstring("")
         return Bitstring(format(int.from_bytes(self._value, "big"), f"0{self.size}b"))
```

Comment:
You could `size` here instead of `self.size` for consistency.

Reply:
Fixed by https://github.com/Componolit/RecordFlux/pull/538/commits/ea7c5f5be480a596ab27e4ac8d2bd6f38eaeb36f.